src/components/HeartbeatBar.vue

```diff
@@ -111,6 +98,47 @@ export default {
             return placeholders.concat(this.beatList.slice(start));
         },
 
+        values() {
+            if (!this.shortBeatList || !this.shortBeatList.length) {
+                return [];
+            }
+
+            const valueObj = {};
+
+            this.shortBeatList.forEach(({ status, time }) => {
+                const date = dayjs(time).format("YYYY-MM-DD");
+
+                if (valueObj[date] === undefined) {
+                    valueObj[date] = 0;
+                }
+
+                let count = 0;
+
+                switch (status) {
+                    case 0:
```

Comment:
Please use the constants instead of hardcoding the status codes for better readability